document/core/text/instructions.rst

```diff
@@ -214,6 +214,92 @@ Lexically, an |Toffset| or |Talign| phrase is considered a single :ref:`keyword
    \end{array}
 
 
+.. index:: atomic memory instruction
+   pair: text format; instruction
+.. _text-instr-atomic-memory:
+
+Atomic Memory Instructions
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _text-atomic-load:
+.. _text-atomic-loadn:
+.. _text-atomic-store:
+.. _text-atomic-storen:
+.. _text-atomic-rmw:
+.. _text-atomic-rmwn:
+
+The offset immediate to atomic memory instructions is optional, and defaults to
+:math:`\T{0}`.
+
+.. math::
+   \begin{array}{llclll}
+   \production{instruction} & \Tplaininstr_I &::=& \dots \\ &&|&
+     \text{i32.atomic{.}load}~~m{:}\Tmemarg_4 &\Rightarrow& \I32.\ATOMICLOAD~m \\ &&|&
```

Comment:
Yeah, this was strange -- I found that without this Mathjax would produce extra space after `.`, probably because it assumes that it is the end of a sentence. I suppose it doesn't do this for `32.` because it thinks its a number?